bin/crew

```diff
@@ -708,7 +708,7 @@ def download
       # Recall repository from cache if requested
       if CREW_CACHE_ENABLED
         if @pkg.git_branch.nil? || @pkg.git_branch.empty?
-          cachefile = CREW_CACHE_DIR + filename + @pkg.git_hashtag + '.tar.xz'
+          cachefile = CREW_CACHE_DIR + filename + @pkg.git_hashtag.gsub('/', '_') + '.tar.xz'
```

Comment:
A git hashtag should only include hex characters.  Not sure why this would be needed unless the git hashtag was used to indicate an upstream/branch.  But then it shouldn't be called a hashtag in that instance.

Reply:
A lot of tags from https://salsa.debian.org/ have this pattern.